write-fonts/Cargo.toml

```diff
@@ -26,7 +26,9 @@ indexmap = "2.0"
 diff = "0.1.12"
 ansi_term = "0.12.1"
 font-test-data = { path = "../font-test-data" }
-read-fonts = { version = "0.16.0", path = "../read-fonts", features = [ "codegen_test"] }
+read-fonts = { version = "0.16.0", path = "../read-fonts", features = [
```

Comment:
weird that this got reformatted, did your editor do this automatically? Not important, I'm just curious..

Reply:
It did. I grabbed https://taplo.tamasfe.dev/ as a vscode extension, and didn't notice that it comes with an auto formatter